articles/stripe-integration-react/index.md

```diff
@@ -0,0 +1,123 @@
+---
+  layout: engineering-education
+  status: publish
+  published: true
+  url: /engineering-education/stripe-integration-react/
+  title: Integrating Stripe with React
+  description: Learn how to create a Stripe account and integrate Stripe Payment with your React project.
+  author: lalith-narayan
+  date: 2020-08-24T00:00:00-10:00
+  topics: [Blockchain][Cryptocurrency][Healthtech]
+  excerpt_separator: <!--more-->
+  images:
+
+    - url: /https://images.app.goo.gl/53Wo8TroqUuzF85i9
```

Comment:
Please upload this as hero.png into your file directory and change the URL accordingly.

Reply:
The link mentioned is broken. I am attaching a new hero image.